vue/src/Views/TurnServers.vue

```diff
@@ -58,7 +60,7 @@
 <script>
 import { Tooltip } from 'nextcloud-vue'
 import debounce from 'debounce'
-import TurnServer from 'Components/TurnServer'
+import TurnServer from '../Components/TurnServer'
```

Comment:
You can still use your custom aliases if you want, you just ened to keep this part of the config on your eslint, it's just a matter of personnal preference :)

Reply:
:man_shrugging: dont care too much, so lets go with the "standard"